src/test/java/io/mosip/mimoto/controller/InjiControllerTest.java

```diff
@@ -639,4 +639,30 @@ public void aidGetIndividualIdTest() throws Exception {
                         .content(JsonUtils.javaObjectToJsonString(requestDTO)))
                 .andExpect(status().isOk());
     }
+
+    @Test
+    public void verifyCredentialTest() throws Exception {
+        File file = ResourceUtils.getFile(ResourceUtils.CLASSPATH_URL_PREFIX + "responses/VerifiableCredential.json");
+        String credential = new String(Files.readAllBytes(file.toPath()));
+
+        Mockito.when(credentialService.verifyCredential(credential))
+                .thenReturn(true)
+                .thenThrow(new ProofTypeNotFoundException("Proof Type available in received credentials is not matching with supported proof terms"));
+
+        this.mockMvc.perform(post("/credentials/verify")
+                        .accept(MediaType.APPLICATION_JSON_VALUE)
+                        .content(credential))
+                .andExpect(status().isOk())
+                .andExpect(jsonPath("$.response", Matchers.is(true)));
+
+
+        this.mockMvc.perform(post("/credentials/verify")
+                        .accept(MediaType.APPLICATION_JSON_VALUE)
+                        .content(credential))
+                .andExpect(status().is5xxServerError())
```

Comment:
Why is this a 5xx code, since it is a checked and catched exception, shouldn't it be 4xx ?